app/models/manageiq/providers/regions.rb

```diff
@@ -21,15 +21,15 @@ def from_source
       end
 
       def additional_regions
-        Settings.dig(:ems, ems_type, :additional_regions).to_h.stringify_keys
+        Settings.dig(:ems, ems_type, :additional_regions).to_hash.stringify_keys
       end
 
       def disabled_regions
         Settings.dig(:ems, ems_type, :disabled_regions).to_a
       end
 
       def ems_type
-        vendor = module_parent.name.sub("ManageIQ::Providers::", "").sub("::", "_").downcase
+        vendor = module_parent.name.sub("ManageIQ::Providers::", "").sub("::", "_").underscore
```

Comment:
NOTE: this was causing an issue with `ManageIQ::Providers::OracleCloud` because it was resulting in `:ems_oraclecloud` and not `:ems_oracle_cloud`.

Short term this resolves the issue but I think we should move this into something like `ExtManagementSystem.ems_settings_name` to allow providers to change this if they have to (e.g. IBM Cloud VPC) all in one place.